x-pack/solutions/observability/plugins/apm/server/routes/service_map/fetch_exit_span_samples.ts

```diff
@@ -100,7 +100,7 @@ async function fetchExitSpanIdsFromTraceIds({
 }) {
   const sampleExitSpans = await apmEventClient.search('get_service_map_exit_span_samples', {
     apm: {
-      events: [ProcessorEvent.span],
+      events: [ProcessorEvent.span, ProcessorEvent.transaction],
```

Comment:
This might resolve the UI issue, but I’m not entirely sure if it’s the right approach.

For otel, this can be true `span.destination.service.resource and processor.event: transaction` . With the otel processor, shouldn't this be translated to a span document instead? 

@crespocarlos what you think?

Reply:
Thank you both for the context @gregkalapos @AlexanderWert.

I just wanted to understand the reasoning behind this change and you’ve covered that.

Not raising it as an issue, just making sure we’re aligned given the change in semantics.